src/erl_optics.erl

```diff
@@ -175,7 +240,10 @@ create_foil() ->
     end.
 
 create_optics() ->
```

Comment:
This doesn't appear to be used by anything, and I don't think it should be exported. Would you mind removing it?